patroni/postgresql/config.py

```diff
@@ -245,7 +245,8 @@ def writeline(self, line):
 
     def writelines(self, lines):
         for line in lines:
-            self.writeline(line)
+            if line is not None: #Yaml import interprets empty strings (for example, comment lines) as None.
+                self.writeline(line)
```

Comment:
The `writeline()` method would be a better place for it.
Also, the check should look like (`six` should be imported):
```python
if isinstance(line, six.string_types):
```

Reply:
@pehrsoderman JFYI